app/templates/login.hbs

```diff
@@ -0,0 +1,141 @@
+<AuApp>
+  <AuMainContainer as |m|>
+    <m.content @scroll={{true}}>
+      <AuContentHeader
+        @titlePartOne="Vlaanderen"
+        @titlePartTwo="is lokaal bestuur"
+      >
+        <img
+          sizes="50vw"
+          src={{root-url
+            "/assets/images/loket-header-1600-85a5251e74ef6ac8491b2f228d2c90da.jpg"
+          }}
+          srcset={{@srcSet}}
+          alt="Foto van een laptop met daarop het vlaanderen logo."
+        />
+      </AuContentHeader>
+
+      <section class="au-o-region-large">
+        <div class="au-o-layout">
```

Comment:
These can be combined into 1 element and  replaced by the `au-o-box au-o-box--large` classes.